crates/matrix-sdk/src/notification_settings/rules.rs

```diff
@@ -222,16 +226,16 @@ impl Rules {
 /// # Arguments
 ///
 /// * `is_encrypted` - `true` if the room is encrypted
-/// * `members_count` - the room members count
-fn get_predefined_underride_room_rule_id(
+/// * `is_one_to_one` - `true` if the room is a direct chat
+pub(crate) fn get_predefined_underride_room_rule_id(
     is_encrypted: bool,
-    members_count: u64,
+    is_one_to_one: bool,
```

Comment:
nit: if you want a boolean here, can you name it `is_direct_message` or `is_dm_here`? It's not clear that a DM only involves 2 people, from the Matrix spec.

Reply:
I recommend `is_dm` instead of `is_dm_here` ^^.

Reply:
Historically, from the client application's point of view, a room is a DM if it's declared as such in the account data, but for push rules, a one-to-one room is one with exactly 2 members. Naming this parameter `is_dm` can lead to confusion, as what we want is not the `is_direct` but for the number of members to be exactly equal to 2.

Reply:
The push rules being returned here use the same `OneToOne` naming, so I think we must not change anything here to avoid confusion.

Reply:
It does check the account data.

Reply:
If you wanted you could use the two enums here too, for consistency.